Update docs related to plugin loading. (#7737)
cakephp/docs: en/plugins.rst

If you install your plugins via ``composer`` or ``bake`` you shouldn't need to
configure class autoloading for your plugins.

> If you create a plugin manually under the ``plugins`` folder then will need to
> tell ``composer`` to refresh its autoloading cache:

Reviewer: ```suggestion
If you create a plugin under the ``plugins`` directory of your application then will need to
tell ``composer`` to refresh its autoloading cache:
```

Author: I used the word "manually" to indicate as opposed to baking, since baking a plugin already runs `composer dumpautoload` now.

Reviewer 2: there is a word missing after "then"
